src/prior_generators/sparse_depth/Tester02.py

```diff
@@ -38,13 +38,13 @@ def clean_tmp_dir(self):
         self.tmp_dirpath.mkdir(parents=True)
         return 
     
-    def save_tmp_data(self, image_paths: List[Path], intrinsics: numpy.ndarray, resolution: tuple = None):
+    def save_tmp_data(self, images: numpy.ndarray, intrinsics: numpy.ndarray, resolution: tuple = None):
```

Comment:
Don't take images as input. Taking image paths is a better option. Will be faster. Please modify the DataGenerator file to pass in image paths instead of reading the images and passing them.

Reply:
It was ambiguous, one file had images and the following file had image paths, so picked one way to go ahead. Fixed it

Reply:
Yeah, it was. Sorry for that